root/static/scripts/edit/utility/toggleEnded.js

```diff
@@ -0,0 +1,24 @@
+// This file is part of MusicBrainz, the open internet music database.
+// Copyright (C) 2017 MetaBrainz Foundation
```

Comment:
You should replace that with your own name (unless you intended to assign copyright to MeB).